src/scripts/api.js

```diff
@@ -0,0 +1,11 @@
+const baseUrl = 'https://mate-api.herokuapp.com';
```

Comment:
we name constants in upper case

```suggestion
const BASE_URL = 'https://mate-api.herokuapp.com';
```